### PR TITLE
adding _r-mutex recipe

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [ ] ~Build number is 0~ `r-base` is requiring 1.* anacondar_1
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Dear All,

to make conda-forge more self-contained I want to the _r-mutex R package which is necessary for installing r-base from conda-forge.
I'm open to discuss if tests are needed because this package resembles a mutex.

This PR resolves the issue https://github.com/conda-forge/r-base-feedstock/issues/83 by mirroring https://github.com/AnacondaRecipes/aggregateR/tree/master/_r-mutex-feedstock

### CODE REVIEW

File: recipes/_r-mutex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  string: anacondar_1
+  number: 1
+
+test:
+  commands:
+    - conda list | grep '_r-mutex'
+
+about:
+  home: https://anaconda.org/r/_r-mutex
+  license: BSD
```
Comment (scopatz) [inline recipes/_r-mutex/meta.yaml]: This needs a `license_file` entry

Comment (fhoehle) [reply recipes/_r-mutex/meta.yaml]: - [ ] add license_file entry

File: recipes/_r-mutex/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+BSD 3-clause license
+Copyright (c) 2015-2019, conda-forge
```
Comment (xhochy) [inline recipes/_r-mutex/LICENSE.txt]: This recipe originates from Anaconda Inc., this should be reflected here.

Comment (fhoehle) [reply recipes/_r-mutex/LICENSE.txt]: Thanks for spotting it! Fixed.

**Pull request merged**